.isort.cfg

```diff
@@ -10,14 +10,12 @@ skip=
 known_first_party = lutris
 sections =
     FUTURE,
-    FUTURE_LIBRARY,
     STDLIB,
-    THIRDPARTY
-    DEPS,
+    THIRDPARTY,
     FIRSTPARTY,
     LOCALFOLDER
-min-similarity-lines=5
+;min-similarity-lines=5
 default_section=THIRDPARTY
-ignore-comments=yes
-ignore-docstrings=yes
-ignore-imports=yes
+;ignore-comments=yes
+;ignore-docstrings=yes
+;ignore-imports=yes
```

Comment:
Remove these lines entirely. These disabled options are no longer supported on isort 5.x.

Reply:
What about min-similarity-lines? Should I delete that too?

Reply:
Yes, support for that was also removed.

Reply:
Right, it's done.